src/core/reopt.jl

```diff
@@ -66,6 +66,9 @@ end
 Solve the model using a `Scenario` or `BAUScenario`.
 """
 function run_reopt(m::JuMP.AbstractModel, s::AbstractScenario)
+	if s.site.CO2_emissions_reduction_min_pct > 0.0 || s.site.CO2_emissions_reduction_max_pct < 1.0
+		@error "To constrain CO2 emissions reduction min or max percentages, the optimal and business as usual scenarios must be run in parallel. Use a version of run_reopt() that takes an array of two models."
```

Comment:
@hdunham should this error be thrown? (And we could create an issue for adding BAU emissions so that a BAU Scenario is not required in the future).

Reply:
Throwing it now. There's a lot of other instances of @error existing throughout the code that should probably be thrown as well.

Reply:
@hdunham do you have this file modified locally? I do not see the `throw` statement.